python/cuml/cluster/CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ add_subdirectory(hdbscan)
 rapids_cython_create_modules(
   CXX
   SOURCE_FILES "${cython_sources}"
-  LINKED_LIBRARIES $<IF:$<BOOL:${SINGLE_GPU}>,${cuml_sg_libraries},${cuml_mg_libraries}>
+  LINKED_LIBRARIES "${cuml_mg_libraries}"
```

Comment:
why is this change needed? This would build the mg targets in singlegpu mode

Reply:
Previously `cuml_mg_libraries` was only defined if `SINGLE_GPU` was `OFF`. [I now define it as equivalent to `cuml_sg_libraries` when `SINGLE_GPU` is `ON` so that each of these targets can link to it unconditionally and trust that the set of libraries is already correct. IIRC there were some typos at one point so I centralized this way as part of resolving some build issues.